cmd/generate.go

```diff
@@ -172,8 +193,6 @@ func updateDiff(config internal.Config, args []string) {
 			log.Panicln(err)
 		}
 	} else {
```

Comment:
Instead of a large else block, return from the func at the end of the if block.

Reply:
it's not part of the PR